arm/Microsoft.Storage/storageAccounts/.parameters/parameters.json

```
@@ -3,7 +3,7 @@
     "contentVersion": "1.0.0.0",
     "parameters": {
         "name": {
-            "value": "sxxazsaweux001"
+            "value": "<<namePrefix>>aweux001"
```

Removed too much I think.
```suggestion
            "value": "<<namePrefix>>azsaweux001"
```